cachi2/core/package_managers/bundler/main.py

```diff
@@ -159,6 +159,7 @@ def _prepare_for_hermetic_build(source_dir: RootedPath, output_dir: RootedPath)
         BUNDLE_CACHE_PATH: "${output_dir}/deps/bundler"
         BUNDLE_DEPLOYMENT: "true"
         BUNDLE_NO_PRUNE: "true"
+        BUNDLE_VERSION: "system"
```

Comment:
**Commit message**:
nitpick: All true except that bundler apparently attempts to do this only with Z-streams, a version mismatch in Y-stream or major leads to a warning only that bundler isn't trying to correct by itself.